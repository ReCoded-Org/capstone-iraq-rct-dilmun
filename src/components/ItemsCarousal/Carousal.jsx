import React from 'react'
import Carousel from 'react-multi-carousel'
import LandingPageCard from '../LandingPageCard'
import 'react-multi-carousel/lib/styles.css'
import '../../App.css'

const responsive = {
  superLargeDesktop: {
    breakpoint: { max: 4000, min: 3000 },
    items: 4,
  },
  desktop: {
    breakpoint: { max: 3000, min: 1024 },
    items: 3,
  },
  tablet: {
    breakpoint: { max: 1024, min: 640 },
    items: 2,
  },
  mobile: {
    breakpoint: { max: 640, min: 0 },
    items: 1,
  },
}

export default function Carousal() {
  return (
    <div>
      <Carousel
        responsive={responsive}
        draggable
        ssr
        infinite
        keyBoardControl
        focusOnSelect
        itemClass="grid justify-items-center p-5"
      >
        <LandingPageCard />
        <LandingPageCard />
        <LandingPageCard />
        <LandingPageCard />
        <LandingPageCard />
      </Carousel>
    </div>
  )
}
