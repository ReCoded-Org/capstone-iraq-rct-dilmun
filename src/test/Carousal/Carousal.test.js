import React from 'react'
import renderer from 'react-test-renderer'
import { BrowserRouter as Router } from 'react-router-dom'
import ItemsCarousal from '../../components/ItemsCarousal'

it('Test: Contact info Card without Any props', () => {
    const tree = renderer
        .create(
            <Router>
                <ItemsCarousal />
            </Router>
        )
        .toJSON()
    expect(tree).toMatchSnapshot()
})
