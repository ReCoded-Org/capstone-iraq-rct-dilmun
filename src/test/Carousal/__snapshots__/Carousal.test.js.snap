// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test: Contact info Card without Any props 1`] = `
<div>
  <div
    className="react-multi-carousel-list  "
  >
    <ul
      className="react-multi-carousel-track "
      onMouseDown={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onMouseMove={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      style={
        Object {
          "overflow": "unset",
          "transform": "translate3d(0px,0,0)",
          "transition": "none",
        }
      }
    />
  </div>
</div>
`;
